examples/platform/change_role.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# *******************************************************
+# Copyright (c) VMware, Inc. 2020-2021. All Rights Reserved.
+# SPDX-License-Identifier: MIT
+# *******************************************************
+# *
+# * DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+# * WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+# * EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+# * WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+# * NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+
+"""Change a user's role from the command line."""
+
+from cbc_sdk.helpers import build_cli_parser, get_cb_cloud_object
+from cbc_sdk.platform import User
+import sys
+
+
+def main():
+    """Main function for the role changing script"""
+    parser = build_cli_parser("Change User's Role")
+    parser.add_argument('email', help='E-mail address of the user to change role of')
+    parser.add_argument('role', help='New role URN for the user')
```

Comment:
Would you consider making this an input based script after the initial args to select one of the permitted roles?

Reply:
Likely something to consider for the other example scripts too if the role argument is not provided

Reply:
What I was thinking was that the role argument would be optional so that if it is not specified then you would print the the permitted roles for the API key so that the user can choose one of them. I really like the concept of this script however a user may not know the role urns so being able to let them see and choose would be helpful. By making the role argument optional a user could also provide the property if they already know the URN

```
What role would you like to configure?
1. View Only
2. Security Analyst I
3. Security Analyst II
```

Reply:
`Grant.get_permitted_roles` only returns the URNs of the roles, not any human-readable text such as you want to display here.  We'd have to change that API call if you wanted to do this.

Reply:
There are other useful properties in the roles model, we should consider how to best to expose that information in the CBC SDK. Would it make sense to rename `get_permitted_roles` to `get_permitted_role_urns` and create an additional function to return the entire role object? Or is there another better option to provide the full role context?

Reply:
https://developer.carbonblack.com/reference/carbon-black-cloud/platform/latest/access-profiles-and-grants/#get-permitted-roles

Reply:
Well, we could go _completely_ down the rabbit hole here and make Role a first-class object within the SDK that can be queried via the `cb.select()` mechanism, but do you want to delay the 1.3 release of the SDK long enough to do this?  To put it another way, what's the _simplest_ thing we can do to satisfy your concerns on this particular item?

Reply:
Capturing the effort to better handle roles in a future story

Reply:
Following discussion during sprint planning, I have renamed `get_permitted_roles` to `get_permitted_role_urns` to leave room for future expansion of the role information we return, by whatever mechanism.  It's not worth holding up the 1.3 release to do anything else right now.